recipe/patches/0018-Use-Intel-LLVM-openmp.patch

```diff
@@ -0,0 +1,31 @@
+From ffbdd8bb8fa38397ad6d7958a8c4ddd80ff738eb Mon Sep 17 00:00:00 2001
+From: Isuru Fernando <isuruf@gmail.com>
+Date: Thu, 31 Jul 2025 12:34:30 -0500
+Subject: [PATCH] Use Intel/LLVM openmp
```

Comment:
The commit message is a bit confusing to me. We're clearly preferring `llvm-openmp` now, right? That's what adding `-openmp:llvm` comes down to. 

Plus MKL itself now comes with the llvm variant: https://github.com/conda-forge/intel_repack-feedstock/pull/107

So I thought we want to avoid all use of `intel-openmp` going forward? Hence my confusion with the commit message.

Reply:
> We're clearly preferring llvm-openmp now, right? That's what adding -openmp:llvm comes down to.

No, it's just a way of saying to cl.exe to codegen LLVM-style symbols, i.e. `kmp_*` instead of the default `vcomp_*` symbols. Both Intel and LLVM supports `kmp_*` files.

> So I thought we want to avoid all use of intel-openmp going forward? 

Yes, but I want to get one build that's correct with intel-openmp before switching. Last PR was not adequate.

Reply:
Thanks for the clarification